### PR TITLE
Feature/add element tests

### PR DESCRIPTION
**Pull request by mialeska**
- Implemented webtests for elements interaction
- Refactored Android NativeApp tests
- Refactored  library screen classes
- Refactored CheckBox and RadioButton elements

### CODE REVIEW

File: Aquality.Appium.Mobile/src/Aquality.Appium.Mobile/Resources/Localization/ru.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
   "loc.platform.name.wrong": "Платформа '{0}' не поддерживается.",
   "loc.application.ready": "Приложение на платформе '{0}' готово...",
   "loc.application.implicit.timeout": "Установка таймаута implicit(неявного) ожидания: '{0}' сек.",
+  "loc.application.android.activity.start": "Стартуем '{0}' активность Android приложения в пакете '{1}'",
```
Comment (sunigos) [inline Aquality.Appium.Mobile/src/Aquality.Appium.Mobile/Resources/Localization/ru.json]: Maybe it's better to say `Старт активности {0}...` ?

File: Aquality.Appium.Mobile/tests/Aquality.Appium.Mobile.Tests/Samples/Android/Web/WebTextBoxTest.cs [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        private const string ValueToSubmit = "quality assurance";
+
+        [Test]
+        public void TextBoxInteraction()
+        {
+            AqualityServices.Application.Driver.Url = "https://wikipedia.org";
+            var txbSearch = AqualityServices.ElementFactory.GetTextBox(By.Id("searchInput"), "Search");
+            txbSearch.State.WaitForClickable();
+            txbSearch.Click();
+            // not yet implemented in dotnet Appium client: Assert.IsTrue(driver.isKeyboardShown(), "Keyboard should be shown when click successful");
```
Comment (sunigos) [inline Aquality.Appium.Mobile/tests/Aquality.Appium.Mobile.Tests/Samples/Android/Web/WebTextBoxTest.cs]: Isn't it better to place `//TODO` to not forget about such things?

**Review by sunigos — APPROVED** [review]

File: Aquality.Appium.Mobile/src/Aquality.Appium.Mobile/Elements/CheckBox.cs [hunk 3]
```diff
@@ -4,44 +4,45 @@
...
         }
 
-        public bool IsChecked
+        public override bool IsChecked
         {
             get
             {
                 LogElementAction("loc.checkbox.get.state");
-                return GetElement().Selected;
+                return base.IsChecked;
```
Comment (knysh) [inline Aquality.Appium.Mobile/src/Aquality.Appium.Mobile/Elements/CheckBox.cs]: I think we olso need LogElementAction for RadioButtonElement
What if you add LogElementAction to base.IsChecked and put ElementType to message pattern?

**Review by knysh — CHANGES_REQUESTED** [review]

File: Aquality.Appium.Mobile/src/Aquality.Appium.Mobile/Elements/CheckBox.cs (hunk 3 above)
Comment (mialeska) [reply Aquality.Appium.Mobile/src/Aquality.Appium.Mobile/Elements/CheckBox.cs]: I believe we should not log this message for callings from other methods like "Toggle", "Check" and "Uncheck". 
There was a comment from @sunigos at java repo about it: https://github.com/aquality-automation/aquality-appium-mobile-java/pull/11#discussion_r395119700 .
Agree that we should probably log it for radiobutton

Comment (mialeska) [reply Aquality.Appium.Mobile/src/Aquality.Appium.Mobile/Elements/CheckBox.cs]: Fixed

File: Aquality.Appium.Mobile/src/Aquality.Appium.Mobile/Resources/Localization/ru.json (hunk 1 above)
Comment (mialeska) [reply Aquality.Appium.Mobile/src/Aquality.Appium.Mobile/Resources/Localization/ru.json]: Fixed

File: Aquality.Appium.Mobile/tests/Aquality.Appium.Mobile.Tests/Samples/Android/Web/WebTextBoxTest.cs (hunk 2 above)
Comment (mialeska) [reply Aquality.Appium.Mobile/tests/Aquality.Appium.Mobile.Tests/Samples/Android/Web/WebTextBoxTest.cs]: Fixed

**Review by knysh — APPROVED** [review]

**Pull request merged**